ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__installation-nightly-4.19.yaml

```diff
@@ -648,34 +648,70 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-azure-ipi-private-sshkey
-- as: azure-ipi-user-assigned-identity-f14
-  cron: 31 21 16,30 * *
+- as: azure-ipi-user-assigned-identity-pre-f14
+  cron: 11 23 7,23 * *
   steps:
     cluster_profile: azure-qe
     env:
+      AZURE_IDENTITY_TYPE_COMPUTE: UserAssigned
+      AZURE_IDENTITY_TYPE_CONTROL_PLANE: UserAssigned
+      AZURE_IDENTITY_TYPE_DEFAULT_MACHINE: UserAssigned
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       BOOT_DIAGNOSTICS: "false"
-      ENABLE_AZURE_IDENTITY_COMPUTE: "true"
-      ENABLE_AZURE_IDENTITY_CONTROL_PLANE: "true"
-      ENABLE_AZURE_IDENTITY_DEFAULT_MACHINE: "true"
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-azure-ipi-identity
-- as: azure-ipi-multi-user-assigned-identity-mini-perm-f28
-  cron: 54 13 29 * *
+- as: azure-ipi-multi-user-assigned-identity-pre-mini-perm-f28
+  cron: 55 7 4 * *
   steps:
     cluster_profile: azure-qe
     env:
+      AZURE_IDENTITY_TYPE_DEFAULT_MACHINE: UserAssigned
       AZURE_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
       AZURE_USER_ASSIGNED_IDENTITY_NUMBER: "2"
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       BOOT_DIAGNOSTICS: "false"
-      ENABLE_AZURE_IDENTITY_DEFAULT_MACHINE: "true"
       FEATURE_GATES: '["MachineAPIMigration=true"]'
       FEATURE_SET: CustomNoUpgrade
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-azure-ipi-identity
+- as: azure-ipi-user-assigned-identity-f14
```

Comment:
```suggestion
- as: azure-ipi-default-identity-f14
```

Reply:
BTW, I guess the jobs setting `AZURE_IDENTITY_TYPE_*` have the same behavior as without setting them, right ?

Reply:
With PR https://github.com/openshift/installer/pull/9735, yes, same behavior.

Reply:
So the job is to test explicit configurations ? If yes, maybe we can lower its frequency ?

Reply:
Done